bool iris_connected
uint16 voltage_24v